modules/build/src/main/scala/scala/build/Artifacts.scala

```diff
@@ -129,12 +129,23 @@ object Artifacts {
         Nil
     }
 
+    val scala2NightlyRepo = Seq(coursier.Repositories.scalaIntegration.root)
+
     val scalaNativeCliDependency =
       scalaNativeCliVersion.map(version =>
         Seq(dep"org.scala-native:scala-native-cli_2.12:$version")
       )
 
-    val allExtraRepositories = maybeSnapshotRepo ++ extraRepositories
+    val scala2NightlyRegex = raw"""2\.(\d+)\.(\d+)-bin-[a-f0-9]*""".r
```

Comment:
Move this regex to `ScalaParse.scala`.

Reply:
it is applied here:
https://github.com/VirtusLab/scala-cli/pull/655#issuecomment-1040123517

Reply:
modules/build/src/main/scala/scala/build/internal/ScalaParse.scala